### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2378,6 +2380,8 @@ chasnge->change, changes,
 chasr->chaser, chase,
 chatacter->character
 chatacters->characters
+chatch->catch
+chatched->caught
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Possibly add chatted as a suggestion.

Does catched exist as a typo?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5654,6 +5659,10 @@ executbale->executable
 executeable->executable
 executeables->executables
 executiong->execution, executing,
+executs->executes
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add execut

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -8302,6 +8313,7 @@ livley->lively
 lke->like
 lmits->limits
 loaader->loader
+loacally->locally
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add loacal

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -14911,6 +14931,8 @@ weant->want, wean,
 weaponary->weaponry
 weas->was
 webiste->website
+wedensday->wednesday
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I think we can capitalise the suggestion correctly can't we.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -13487,6 +13505,8 @@ syste->system
 systemwindiow->systemwindow
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can we add "system window" as the first suggestion.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]:
```
$  echo "catched" | codespell -
1: catched
	catched ==> caught
```

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I was being lazy

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: > Possibly add chatted as a suggestion.  

I think this is a bit of a strech. Perhaps @larsoner can weigh in?

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @larsoner opinion?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: It's quite a few mistyped letters I'd agree, but if you're going for people misspelling things, as you are with the end result here being caught, I don't think it's a huge leap. If they're unsure of how to spell it, we ought to offer the other word they might be trying to spell, so they don't get even more confused.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: yes we can

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: No opinion either way

**Comment by luzpaz** [comment]
Added requested refinements in 2475bae and 2ea7637
@peternewman please review
and thank you.

**Comment by luzpaz** [comment]
@peternewman please review revisions. thanks

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -5743,6 +5753,8 @@ exort->export
 exorted->exported, extorted, exerted,
 exoskelaton->exoskeleton
 expalin->explain
+expanation->explanation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Perhaps add expansion as a suggestion

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -5743,6 +5753,8 @@ exort->export
...
+expanations->explanations
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Possibly and expansions

**Comment by luzpaz** [comment]
done in e278921

**Pull request merged**